docs-search/index/src/Docs/Search/IndexBuilder.purs

```diff
@@ -72,15 +70,16 @@ type Config =
   , generatedDocs :: String
   , workspacePackages :: Set PackageName
   , moduleGraph :: Graph.ModuleGraphWithPackage
+  , log :: String -> Aff Unit
+  , die :: String -> Aff Unit
```

Comment:
`IndexBuilder` was logging directly to console with no indirection at all. I added these parameters so that it would respect the `--quiet` option.

Looking at the larger picture, I don't understand why this is a separate project and not part of Spago proper. It's not used from anywhere else, only from the `docs` command implementation.

Reply:
It was born as a separate experiment, and was integrated only recently in Spago's codebase. It could be used independently from the command line as well (functionality that we ought to restore as a Spago command)